### PR TITLE
[Takeover + Engage page] Yahoo (JP)

### PR DESCRIPTION
**Pull request by solazio**
## Done

- Add Yahoo takeover
- Add Yahoo engage page

## QA

- Go to [http://0.0.0.0:8012/](http://0.0.0.0:8012/) & [/engage/yahoo](http://0.0.0.0:8012/engage/yahoo)
- Compare to [brief](https://docs.google.com/document/d/16DSb-YOZsC6HqcF06OjylXRilUIyxna0Ws6E6fjPwRw/edit) and [design](https://github.com/canonical-web-and-design/web-squad/issues/1360)

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/1361

### CODE REVIEW

**Comment by solazio** [comment]
@SirSamTumless and @sowasred2012 this needs the return url marketo information.

File: webapp/blueprint.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
     "jp_website",
     __name__,
     template_folder="/templates",
-    static_folder="/static",
+    static_folder="/static"
```
Comment (deadlight) [inline webapp/blueprint.py]: I think that coding standards are to keep the last comma

**Comment by deadlight** [comment]
QA looks good other than the already-noted missing marketo config

**Comment by SirSamTumless** [comment]
Hi @here, Marketing got some feedback from the JP PR company. Let me know if it makes sense?

So the main headline would be... but with the amend (those two characters) copied below. 

Yahoo! JAPAN、Canonicalとプライベートクラウドの構築で協働

>It says it's a private cloud, but the PDF says it's a public cloud. Also, "協働" might be technically correct in a dictionary, but I've never seen that word in my life to be honest. "協業" would be a more general term.

And then the sub-head would be (corrected version)... 

Yahoo! JAPANが、MAASとJujuを使用して2万VM用のOpenStackと1PBのCephストレージを3リージョンに展開した事例を紹介

**Comment by SirSamTumless** [comment]
@deadlight That's good to review.

File: templates/engage/yahoo.html [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+      <p>ヤフー株式会社は、Yahoo!とソフトバンクの合弁事業として設立された、国内で最も人気の高いインターネット広告、検索エンジン、ECサイトのひとつで、6,000人を超える従業員を擁しています。</p>
+      <p>非常に大規模で多数のユーザーを抱えていることから、ヤフーは自社のIaaS（Infrastructure as a Service：サービスとしてのインフラストラクチャ）ソリューションとOSS分散ストレージソリューションを構築するために外部の協力を必要としました。2013年、ヤフーはCanonicalに協力を求め、それ以降両社は協力関係を育んできました。</p>
+      <blockquote class="p-pull-quote">
+        <p class="p-pull-quote__quote">予期しない障害発生時はOSレイヤーまで調査しなければなりませんが、その際にプロフェッショナルへ問い合わせできるのは大きい。ここまで手厚く対応してもらえるのはCanonicalだけだと思います。</p>
+        <cite class="p-pull-quote__citation">ヤフー株式会社 システム統括本部 サイトオペレーション本部 クラウド開発リーダーの阪田氏</cite>
+      </blockquote>
+      <p>ヤフーでは、Ubuntuに加えCanonicalが提供する幅広い運用ツールを使用して、大規模なIaaS環境の構築と運用に成功しています。また同時にCAPEXとOPEXの両方のコストを削減しています。</p>
+      <p>このユーザー事例では、以下について紹介しています。</p>
+      <ul>
+        <li><a class="p-link--external" href="https://maas.io/">MaaS</a>と<a class="p-link--external" href="https://jaas.ai/">Juju</a>を使用して、3リージョンにまたがる1PBのCephストレージを用いた<a class="p-link--external" href="https://ubuntu.com/openstack">OpenStack</a>プライベートクラウドを展開。</li>
```
Comment (nobuto-m) [inline templates/engage/yahoo.html]: The following diff should be also applicable here:
https://github.com/canonical-web-and-design/jp.ubuntu.com/pull/157/commits/591355b895f476a13bb90466310cf7f839b555d1#diff-fb5aa1cd1261d08d02db6f7dc314d9abL13

Expected text is:
"Yahoo! JAPANが、MAASとJujuを使用して2万VM用のOpenStackと1PBのCephストレージを3リージョンに展開した事例を紹介"

Comment (solazio) [reply templates/engage/yahoo.html]: @nobuto-m I pushed another commit to add your suggestion.

Comment (nobuto-m) [reply templates/engage/yahoo.html]: Looks good to me. Thanks!

A knitpicking, but replacing `MaaS` by `MAAS` would be nicer.

**Comment by SirSamTumless** [comment]
Here the link for the thank-you page. https://pages.ubuntu.com/CStudy_YahooJP_TY.html

**Comment by SirSamTumless** [comment]
<script src="//app-sjg.marketo.com/js/forms2/js/forms2.min.js"></script>
<form id="mktoForm_3427"></form>
<script>MktoForms2.loadForm("//app-sjg.marketo.com", "066-EOV-335", 3427);</script>

**Comment by SirSamTumless** [comment]
@solazio Alex flagged that the thank you page wasn't in Japanese - which is a problem.

I will grab Peter when he's free to discuss.

**Pull request merged**